types/mustache/index.d.ts

```diff
@@ -198,6 +201,9 @@ declare class MustacheWriter {
      *
      * @param template
      * The template to parse.
+     *
+     * @param tags
+     * The tags to use.
      */
     parse(template: string, tags?: any): any;
```

Comment:
I think you missed adjusting the type of tags here.
Looks fine to me besides that - good job :smile: